sql/moz-fx-data-marketing-prod/ga_derived/firefox_whatsnew_summary_v1/query.sql

```diff
@@ -0,0 +1,34 @@
+WITH wnp_visits AS (
+  SELECT
+    date,
+    visit_identifier,
+    hit_number,
+    bounces,
+    first_interaction,
+    TRIM(page_path_level1, '/') AS locale,
+    page_level_2 AS version,
+    country,
+    IF(hit_number = first_interaction AND bounces = 1, TRUE, FALSE) AS is_bounce
+  FROM
+    `moz-fx-data-marketing-prod.ga_derived.www_site_hits_v1`
+  WHERE
+    date = @submission_date
+    AND page_name LIKE '/firefox/%/whatsnew%'
+    AND hit_type = 'PAGE'
+    AND page_level_1 = 'firefox'
+    AND page_level_3 = 'whatsnew'
+)
+SELECT
+  date,
+  country,
+  locale,
+  version,
+  COUNT(DISTINCT visit_identifier) AS visits,
+  COUNT(DISTINCT CASE WHEN is_bounce = TRUE THEN visit_identifier END) AS bounces
+FROM
+  wnp_visits
+GROUP BY
+  1,
+  2,
+  3,
+  4
```

Comment:
@soGaussian as I promised during the dashboard review, here is some proposed code that:
- Excludes views for invalid version numbers which would have been shown "_Sorry, we can’t find that page_" rather than an actual "what's new" page.
- Uses the [norm.browser_version_info](https://github.com/mozilla/bigquery-etl/blob/main/sql/mozfun/norm/browser_version_info/udf.sql) function to get more specific version info, which will allow the results to be filtered by a particular major version number or whether the version was a major release.
```suggestion
WITH wnp_visits AS (
  SELECT
    date,
    visit_identifier,
    TRIM(page_path_level1, '/') AS locale,
    page_level_2 AS version,
    mozfun.norm.browser_version_info(page_level_2) AS version_info,
    country,
    IF(hit_number = first_interaction AND bounces = 1, TRUE, FALSE) AS is_bounce
  FROM
    `moz-fx-data-marketing-prod.ga_derived.www_site_hits_v1`
  WHERE
    date = @submission_date
    AND hit_type = 'PAGE'
    -- Match page paths like "/{locale}/firefox/{version}/whatsnew/..."
    -- Version regular expression is adapted from https://github.com/mozilla/bedrock/blob/main/bedrock/releasenotes/__init__.py
    AND page_level_1 = 'firefox'
    AND REGEXP_CONTAINS(page_level_2, r'^\d{1,3}(\.\d{1,3}){1,3}((a|b(eta)?)\d*)?(pre\d*)?(esr)?$')
    AND page_level_3 = 'whatsnew'
)
SELECT
  date,
  country,
  locale,
  version,
  version_info.major_version,
  version_info.minor_version,
  version_info.patch_revision,
  version_info.is_major_release,
  COUNT(DISTINCT visit_identifier) AS visits,
  COUNT(DISTINCT CASE WHEN is_bounce = TRUE THEN visit_identifier END) AS bounces
FROM
  wnp_visits
GROUP BY
  1,
  2,
  3,
  4,
  5,
  6,
  7,
  8
```

Reply:
For posterity, I got to the version regular expression from [https://github.com/mozilla/bedrock/blob/main/bedrock/releasenotes/__init__.py](https://github.com/mozilla/bedrock/blob/6fb6f4a8954a456af022358d70b00a52690819c3/bedrock/releasenotes/__init__.py#L6) based on the [code used to route people to the Firefox "what's new" pages in Bedrock](https://github.com/mozilla/bedrock/blob/6fb6f4a8954a456af022358d70b00a52690819c3/bedrock/firefox/urls.py#L85).